ci-operator/step-registry/cucushift/upgrade/cpou/set-edge/3hops/cucushift-upgrade-cpou-set-edge-3hops-ref.yaml

```diff
@@ -0,0 +1,16 @@
+ref:
+  as: cucushift-upgrade-cpou-set-edge-3hops
```

Comment:
I think this step also can be reused in chain upgrade, so maybe we can rename it to `cucushift-upgrade-set-edge-3hops`

Reply:
Relocated/renamed the step.